hardware_interface/include/hardware_interface/components/joint.hpp

```diff
@@ -131,7 +131,7 @@ class Joint
   HARDWARE_INTERFACE_EXPORT
   return_type set_command(const std::vector<double> & command);
 
-  /**
+  /*
```

Comment:
This is probably a mistake. 
```suggestion
  /**
```